content/olaVeaMail/2022/2022-01-25-wrong/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: This Sunday’s Skill-Builder-Session went WRONG!
+emojii: ⛵️
+---
+
+Ship Ahoy Skill-Builder!
```

Comment:
No need for this line, the code adds it.

Reply:
Ship Ahoy Skill-Builder?

Reply:
Both Skip Ahoy Skill-Builder and the emoji